build.sbt

```diff
@@ -48,6 +48,14 @@ lazy val commonJacocoReportSettings: JacocoReportSettings = JacocoReportSettings
 lazy val commonJacocoExcludes: Seq[String] = Seq(
 )
 
+enablePlugins(FlywayPlugin)
+flywayUrl := FlywayConfiguration.flywayUrl
+flywayUser := FlywayConfiguration.flywayUser
+flywayPassword := FlywayConfiguration.flywayPassword
+flywayLocations := FlywayConfiguration.flywayLocations
+flywaySqlMigrationSuffixes := FlywayConfiguration.flywaySqlMigrationSuffixes
+libraryDependencies ++= flywayDependencies
+
```

Comment:
`database` module is not present. Also isn't it overlapping the modules `database` and `examples`?

Reply:
good point, an overlook on my side about missing `database` - in terms of examples, I didn't care about them here in this PR, but why not, I'll think about deleting / merging or moving them.

Reply:
based on the content of those tests in examples/, I decided to drop them completely. All the tests there are ignored, it refers to `menas` DB, and objects that are not in this repo, and those individual functions that would be tested are basically nicely covered in actual tests in Slick.

Reply:
Good, no objections